docs/deployment/customization/custom_buttons-Reference.md

```diff
@@ -0,0 +1,72 @@
+# Custom Buttons for Data Tables
```

Comment:
i think we need to make this more specific.   custom_buttons_json doesn't imply group_comparison.   i guess ```group_comparison_custom_buttons```?

Reply:
Ok, as to exactly what name: technically, it may appear anywhere that the CopyDownloadControls appear (with ShowDownload==true). I couldn't find an official name for the tables where these controls appear in this context, so I have been calling them "group comparison", but realize it could be a misnomer.  In the RFC, we also used the term "LazyMobX tables", but that doesn't seem right either.

@alisman preference?
A) download_custom_buttons_json
B) group_comparison_custom_buttons_json
C) data_custom_buttons_json

Reply:
using download_custom_buttons_json